tools/spell_checker/dictionary.txt

```diff
@@ -2985,7 +2985,7 @@ unlivable
 unlively
 unlives
 unlooted
-unlubricated
```

Comment:
Why remove this word?

Reply:
Unintended, might've been a rebase issue. Just re-added it.